airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/common.py

```diff
@@ -92,12 +93,15 @@ def should_retry_api_error(exc):
             unknown_error = exc.api_error_subcode() == FACEBOOK_UNKNOWN_ERROR_CODE
             connection_reset_error = exc.api_error_code() == FACEBOOK_CONNECTION_RESET_ERROR_CODE
             server_error = exc.http_status() == http.client.INTERNAL_SERVER_ERROR
+            pattern = r"Cannot include .* in summary param because they weren't there while creating the report run."
+            cannot_include_transient_error = exc.http_status() == http.client.BAD_REQUEST and re.search(pattern, exc.api_error_message())
```

Comment:
Can we document the reasoning why we think those bad requests are retryable? You have describe the source of this information [here](https://github.com/airbytehq/oncall/issues/4951#issuecomment-2052067259) but it would be nice to have this directly in the code so that everyone has access to the logic behind it

Reply:
updated with description